src/route/accesskeys/components/AccessKeyDetails.tsx

```diff
@@ -95,6 +87,25 @@ export default function AccessKeyDetails(Props: {
           </Grid>
           <Grid item xs={12}>
             <Grid container justifyContent="space-evenly" alignItems="center">
+              <Grid item xs={3} style={styles.centerStyle}></Grid>
+              <Grid item xs={7} style={styles.centerStyle}>
+                <Typography
+                  style={{
+                    minWidth: '25vw',
+                    marginBottom: '0.5em',
+                    marginTop: '0.5em',
+                    marginRight: '35rem',
+                  }}
+                >
+                  <Button
```

Comment:
need to include `rel="noopener noreferrer"` on the Button so users stay more anonymous.